src/AzureExtension/DevBox/DevBoxProvider.cs

```diff
@@ -143,6 +143,15 @@ public async Task<IEnumerable<IComputeSystem>> GetDevBoxesAsync(IDeveloperId dev
         // update the cache every time we retrieve new Dev Boxes. This is used so in the creation flow we don't need

```

Comment:
To fix your build Issue, if you keep the changes to this method I think you only need to remove this line:

https://github.com/microsoft/DevHomeAzureExtension/blob/13bed06622b878023931a666471581e61e7bb96c/test/AzureExtension/DevBox/DevBoxTests.cs#L41

and this line:

https://github.com/microsoft/DevHomeAzureExtension/blob/13bed06622b878023931a666471581e61e7bb96c/test/AzureExtension/DevBox/DevBoxTests.cs#L103

try running the tests in visual studio, I think they might pass after that

Reply:
Thank you for this. Save me a lot of time. ❤️